docs/tutorial/t-relate-kafka.md

```diff
@@ -226,8 +230,8 @@ Note that the `kafka-test-app` charm can also similarly be used to consume messa
 juju config kafka-test-app topic_name=test_kafka_app_topic role=consumer consumer_group_prefix=cg
 ```
 
-After configuring the Kafka Test App, just relate it again with the Kafka charm. This will again create a new user and start the consumer process. 
+After configuring the Apache Kafka Test App, just relate it again with the Apache Kafka charm. This will again create a new user and start the consumer process. 
```

Comment:
```suggestion
After configuring the Apache Kafka Test App, just relate it again with the Charmed Apache Kafka. This will again create a new user and start the consumer process. 
```